src/sumo/wrapper/_logging.py

```diff
@@ -0,0 +1,25 @@
+import logging
+from datetime import datetime
+
+
+class LogHandlerSumo(logging.Handler):
+    def __init__(self, sumoClient):
+        logging.Handler.__init__(self)
+        self._sumoClient = sumoClient
+        return
+
+    def emit(self, record):
+        dt = datetime.utcnow().replace(microsecond=0).isoformat() + "Z"
+        json = {
+            "severity": record.levelname,
+            "message": record.getMessage(),
+            "timestamp": dt,
+            "source": record.name,
+            "pathname": record.pathname,
+            "funcname": record.funcName,
+            "linenumber": record.lineno,
+        }
+        self._sumoClient.post("/message-log/new", json=json)
+        return
+
+    pass
```

Comment:
this `pass` should not pass 🧙

Reply:
The pass passes; it's just a way of expicitly ending the class definition (a minor act of defiance of the stupidity that is indentation-based program structure).